src/murmur/Server.cpp

```diff
@@ -55,6 +55,16 @@
 
 #define UDP_PACKET_SIZE 1024
 
+#ifdef Q_OS_WIN
```

Comment:
I think I didn't want to unconditionally do the reinterpret_cast if I don't have to. Though admittedly at first I thought I could get away with a single of these macros...it got uglier...I could argue that when int doesn't always have to be 32bit (while DWORD does) but well....

I guess the reinterpret is cleaner....Fixed in 4ad0b01